### DISCUSSION TITLE
Storefront Categories sidebox flyout/centrebox dropdown

### DISCUSSION THREAD

**Discussion by torvista** [zencart/zencart#6639] [Ideas]
As result of my tail-chasing iterative efforts and difficulties, here’s something to discuss.

Category sidebox: a flyout option seems a pretty common feature to offer.

Subsequently, it would seem logical to leverage the same code to offer a single-column option of that sidebox: as a dropdown flyout.

Further, once done for RClassic, it needs to be ported to Bootstrap4 template...maybe the mobile burger menu is already suitable.

But, I believe Bootstrap5 has widgets for this sort of functionality already.

Maybe the correct approach would be to work backwards, bearing in mind/using the BS5 implementation to ease the future BS5 template.

Having all these aspects in mind means it does not seem logical for anyone to spend time reworking any individual/sticking-plaster solutions into piecemeal PRs without having a overall cunning plan decided upon....

**Comment by barco57** [comment]
A cunning plan you say.... hmmmmm
